src/tools/pub/cmd/pub-deps.md

```diff
@@ -47,7 +47,7 @@ markdown_converter 0.0.0
 
 ## Options
 
-For options that apply to all pub commands, see
+For options that apply to all dart pub commands, see
```

Comment:
No need to specify "dart pub" here, just "pub commands" is fine.This will match the other pub command pages and technically they apply to the flutter sub-command as well.